src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
 document.addEventListener('click', e => {
-  // write code here
+  const item = e.target;
+
+  if (item !== wall) {
+    return;
+  }
+
+  let coordX = e.offsetX - spider.clientWidth / 2;
```

Comment:
Replace 'let' with 'const' for variables 'coordX' and 'coordY', as their values are not being reassigned.